configs/FDRC-FLYDRAGONF722_V2.2.config

```diff
@@ -1,17 +1,17 @@
-# Rotorflight / STM32F7X2 (S7X2) 4.3.0 Jan  9 2024 / 11:09:54 (norevision) MSP API: 12.3
+# Rotorflight / STM32F7X2 (S7X2) 4.3.0 May 15 2024 / 09:06:57 (05570fe) MSP API: 12.6
 
-board_name FLYDRAGONF722M_V2
+board_name FLYDRAGONF722_V2.2
```

Comment:
Sorry but you can't use a dot in the board name. (nor dash).
Please replace the dot with an underscore, or use another name altogether.

Reply:
Oh, sorry, I'll change that.